src/chi-vue/src/views/TooltipView.vue

```diff
@@ -1,114 +1,101 @@
+    <div class="chi-grid">
+      <div class="chi-col -w--12">
+        <div class="-d--flex -w--100 -justify-content--center">
+          <Tooltip message="Top Tooltip message" color="light">
+            <button class="chi-button -outline -light">Top</button>
+          </Tooltip>
         </div>
-        <div class="chi-col -w--6">
-          <div class="-d--flex -w--100 -justify-content--center">
-            <Tooltip class="-mt--3" message="Left Tooltip message" position="left" color="light">
-              <button class="chi-button -outline -light">Left</button>
-            </Tooltip>
-          </div>
+      </div>
+      <div class="chi-col -w--6">
+        <div class="-d--flex -w--100 -justify-content--center">
+          <Tooltip class="-mt--3" message="Left Tooltip message" position="left" color="light">
+            <button class="chi-button -outline -light">Left</button>
+          </Tooltip>
         </div>
-        <div class="chi-col -w--6">
-          <div class="-d--flex -w--100 -justify-content--center">
-            <Tooltip class="-mt--3" message="Right Tooltip message" position="right" color="light">
-              <button class="chi-button -outline -light">Right</button>
-            </Tooltip>
-          </div>
+      </div>
+      <div class="chi-col -w--6">
+        <div class="-d--flex -w--100 -justify-content--center">
+          <Tooltip class="-mt--3" message="Right Tooltip message" position="right" color="light">
+            <button class="chi-button -outline -light">Right</button>
+          </Tooltip>
         </div>
-        <div class="chi-col -w--12">
-          <div class="-d--flex -w--100 -justify-content--center">
-            <Tooltip class="-mt--3" message="Bottom Tooltip message" position="bottom" :color="color">
-              <button class="chi-button -outline -light">Bottom</button>
-            </Tooltip>
-          </div>
+      </div>
+      <div class="chi-col -w--12">
+        <div class="-d--flex -w--100 -justify-content--center">
+          <Tooltip class="-mt--3" message="Bottom Tooltip message" position="bottom" :color="color">
+            <button class="chi-button -outline -light">Bottom</button>
+          </Tooltip>
         </div>
       </div>
     </div>
+  </div>
 
-    <div class="tooltip-update -py--5">
-      <button class="chi-button" @click="() => toggleColor()">Toggle Color</button>
-      <select class="chi-button -ml--1" @change="(e) => changePosition(e)">
-        <option value="top">Top</option>
-        <option value="right">Right</option>
-        <option value="bottom">Bottom</option>
-        <option value="left">Left</option>
-      </select>
-      <Tooltip class="-mt--3 -w--25" message="Dynamic Tooltip message" :position="position" :color="color">
-        <button class="chi-button -w--100" selected>Dynamic tooltip</button>
-      </Tooltip>
-    </div>
+  <div class="tooltip-update -py--5">
+    <button class="chi-button" @click="toggleColor">Toggle Color</button>
```

Comment:
Toggle color is not working for me:

<img width="249" alt="image" src="https://github.com/CenturyLink/Chi/assets/148064753/b2817195-848f-4cc6-b9db-4db02e9096d6">

Class also appears to not have any effect... does it happen to you?